### PR TITLE
VIH-7894 Fix panel member don't stop screen share when resume or start hearing

### PR DESCRIPTION
**Pull request by ZJianWang**
### JIRA link (if applicable) ###

VIH-7894 

### Change description ###

cleanup presentation stream before transfer from WR to conference

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts [hunk 1]
```diff
@@ -75,7 +76,7 @@ import { NotificationToastrService } from '../../services/notification-toastr.se
 import { RoomClosingToastrService } from '../../services/room-closing-toast.service';
 import { ClockService } from 'src/app/services/clock.service';
 
-describe('WaitingRoomComponent EventHub Call', () => {
+fdescribe('WaitingRoomComponent EventHub Call', () => {
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts:79]: Take out fdescribe

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts [hunk 2]
```diff
@@ -51,7 +51,8 @@ import {
     router,
     userMediaService,
     userMediaStreamService,
-    videoWebService
+    videoWebService,
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts:54]: Is this no longer used?

Comment (ZJianWang) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts:54]: I added videoCallService, as for videoWebService I guess there is somewhere used in the test

Comment (FarronF) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts:54]: Yeah I see. The wrong one being used on Line 121 threw me off

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts [hunk 3]
```diff
@@ -383,7 +383,7 @@ describe('WaitingRoomComponent EventHub Call', () => {
         tick();
         flushMicrotasks();
 
-        expect(videoCallService.stopScreenShare).toHaveBeenCalledTimes(1);
+        expect(videoCallService.stopScreenShare).toHaveBeenCalled();
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts:386]: I think we should have the times called here. Was it just incrementing the call with each test? You could do

`videoCallService.stopScreenShare.calls.reset();`

in a beforeEach()

**Review by FarronF — APPROVED** [review]

**Pull request merged**